src/vstest.console/InProcessVsTestConsoleWrapper.cs

```diff
@@ -0,0 +1,1101 @@
+        _testPlatformEventSource.TranslationLayerExecutionStart(
+            0,
+            0,
+            testCaseList.Count,
+            runSettings ?? string.Empty);
+
+        try
+        {
+            TestRequestManager?.ResetOptions();
+
+            var testRunPayload = new TestRunRequestPayload
+            {
+                TestCases = testCases.ToList(),
+                RunSettings = runSettings,
+                TestPlatformOptions = options,
+                TestSessionInfo = testSessionInfo
+            };
+
+            TestRequestManager?.RunTests(
+                testRunPayload,
+                customTestHostLauncher: null,
+                new RunHandlerToEventsRegistrarAdapter(testRunEventsHandler),
+                new ProtocolConfig { Version = _highestSupportedVersion });
+        }
+        catch (Exception ex)
+        {
+            EqtTrace.Error("InProcessVsTestConsoleWrapper.RunTests: Exception occurred: " + ex);
+            var testRunCompleteArgs = new TestRunCompleteEventArgs(null, false, true, ex, null, null, TimeSpan.MinValue);
+
+            testRunEventsHandler.HandleLogMessage(TestMessageLevel.Error, ex.ToString());
+            testRunEventsHandler.HandleTestRunComplete(testRunCompleteArgs, null, null, null);
+        }
+
+        _testPlatformEventSource.TranslationLayerExecutionStop();
+    }
+
+    /// <inheritdoc/>
+    public void RunTestsWithCustomTestHost(
+        IEnumerable<string> sources,
+        string? runSettings,
+        ITestRunEventsHandler testRunEventsHandler,
+        ITestHostLauncher? customTestHostLauncher)
+    {
+        RunTestsWithCustomTestHost(
+            sources,
+            runSettings,
+            options: null,
+            testRunEventsHandler,
+            customTestHostLauncher);
+    }
+
+    /// <inheritdoc/>
+    public void RunTestsWithCustomTestHost(
+        IEnumerable<string> sources,
+        string? runSettings,
+        TestPlatformOptions? options,
+        ITestRunEventsHandler testRunEventsHandler,
+        ITestHostLauncher? customTestHostLauncher)
+    {
+        RunTestsWithCustomTestHost(
+            sources,
+            runSettings,
+            options,
+            testSessionInfo: null,
+            testRunEventsHandler,
+            customTestHostLauncher);
+    }
+
+    /// <inheritdoc/>
+    public void RunTestsWithCustomTestHost(
+        IEnumerable<string> sources,
+        string? runSettings,
+        TestPlatformOptions? options,
+        TestSessionInfo? testSessionInfo,
+        ITestRunEventsHandler testRunEventsHandler,
+        ITestHostLauncher? customTestHostLauncher)
+    {
+        var sourceList = sources.ToList();
+        _testPlatformEventSource.TranslationLayerExecutionStart(
+            1,
+            sourceList.Count,
+            0,
+            runSettings ?? string.Empty);
+
+        try
+        {
+            TestRequestManager?.ResetOptions();
+
+            var shouldLaunchTesthost = true;
```

Comment:
I don't see any other assignment of this `true` why is it here?

Reply:
Not fixed